torch/_meta_registrations.py

```diff
@@ -1349,6 +1349,95 @@ def pool2d_shape_check(
     )
 
 
+@register_meta(aten.max_pool2d_with_indices_backward.default)
+def meta_max_pool2d_with_indices_backward(
+    grad_output, self, kernel_size, stride, padding, dilation, ceil_mode, indices
+):
+    # Reference: aten/src/ATen/native/DilatedMaxPool2d.cpp
+    check(
+        len(kernel_size) == 1 or len(kernel_size) == 2,
```

Comment:
nit. 
```suggestion
        len(kernel_size) in (1, 2),
```
same below.